### PR TITLE
UEFI Secure Boot improvements for ARM (jsc#SLE-15822)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

This update improves UEFI mentionings in respect to the ARM architecture

### PR creator: Are there any relevant issues/feature requests?

* jsc#SLE-15822

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(if applicable to 15 SP4 _only_ -- do not merge yet!)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/libvirt_guest_installation.xml [hunk 1]
```diff
@@ -461,12 +461,18 @@ network=vnet_nated</screen>
       <command>virt-install</command> command.
      </para>
      <para>
-      Secure boot will be used automatically when setting up a new VM
+      &uefisecboot; will be used automatically when setting up a new VM
       with OVMF. To use a specific firmware, use <option>--boot
```
Comment (lproven) [inline xml/libvirt_guest_installation.xml:465]:
```suggestion
      with OVMF. To use specific firmware, use <option>--boot
```

File: xml/vt_installation.xml [hunk 2]
```diff
@@ -275,16 +275,15 @@
   </para>
   <para>
    The firmware used by virtual machines is auto-selected. The auto-selection
-   is based on the *.json files in the
-   <package>qemu-ovmf-<replaceable>ARCH</replaceable></package> package. The
+   is based on the *.json files in the firmware package described above. The
```
Comment (lproven) [inline xml/vt_installation.xml:278]: Should we use a wildcard file spec in the text like this?

File: xml/vt_installation.xml [hunk 3]
```diff
@@ -275,16 +275,15 @@
...
    &libvirt; &qemu; driver parses those files when loading so it knows the
-   capabilities of the various types of firmware. Then when the user selects the type
-   of firmware and any desired features (for example, support for secure boot),
-   &libvirt; will be able to find a firmware that satisfies the user's
-   requirements.
+   capabilities of the various types of firmware. Then when the user selects
+   the type of firmware and any desired features (for example, support for
+   &uefisecboot;), &libvirt; will be able to find a firmware that satisfies the
```
Comment (lproven) [inline xml/vt_installation.xml:282]:
```suggestion
   &uefisecboot;), &libvirt; will be able to find a firmware file that satisfies the
```

Comment (lproven) [reply xml/vt_installation.xml:282]: "Firmware", like "software" and "hardware", is uncountable.

**Review by lproven — CHANGES_REQUESTED** [review]

File: xml/vt_installation.xml (hunk 2 above)
Comment (tbazant) [reply xml/vt_installation.xml:278]:
```suggestion
   is based on the JSON files in the firmware package described above. The
```

**Review by tbazant — COMMENTED** [review]
included Liams suggestions

**Review by lproven — APPROVED** [review]

**Pull request merged**